src/charm.py

```diff
@@ -298,27 +298,65 @@ def _update_config(self) -> None:
         config_yaml = yaml.safe_dump(config)
         config_hash = sha256(config_yaml)
 
-        logger.debug(
-            "config changed" if config_hash != self._stored.config_hash else "no change",
-        )
+        if config_hash == self._stored.config_hash:
+            logger.debug("no change in config")
+            return
+        else:
+            logger.debug("config changed")
```

Comment:
Because we have a `return`, the `else` is not necessary

Reply:
Removed. Thanks!